ubcpi/static/html/ubcpi.html

```diff
@@ -65,6 +65,8 @@ <h4 class="other-answer" ng-if="options[answer.option].text">Student Answer: {{o
 
                 <label class="ubcpi-label" for="rationale">Explain to other students why you chose this answer (Required):</label>
                 <textarea class="ubcpi-field ubcpi-rationale" id="rationale" name="rationale" data-ng-model="rc.rationale" required ng-minlength="{{rationale_size.min}}" ng-maxlength="{{rationale_size.max}}"></textarea>
+                <p class="ubcpi-rationale-warning" ng-if="(rationale_size.max - rc.rationale.length) <= (rc.rationale.length/10)">You are approaching the limit of {{rationale_size.max}} characters for this answer. Characters left: {{rationale_size.max - rc.rationale.length}}</p>
```

Comment:
There might be a problem when the rationale size is equal or larger than max size. When they are equal, the statement is not true ("you are approaching..."). And when larger, the number for characters left will go negative. So you might add additional condition in ng-if to hide this message when the size exceed the max. 

Also the logic in ng-if is wrong, it should be rationale_size.max/10 instead of rc.rationale.length/10